DDR4 OC Guide.md

```diff
@@ -365,7 +368,8 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
    | tWTRS tWTRL | 4 12 | 4 8 | - |
    | tRTP | 12 | 10 | 8 |
    | tCWL<sup>1</sup> | tCL | tCL - 1 | tCL - 2 |
-   * On Intel, tWTRS/L should be left on auto and controlled with tWRRD_dg/sg respectively. Dropping tWRRD_dg by 1 will drop tWTRS by 1. Likewise with tWRRD_sg. Once they're as low as you can go, manually set tWTRS/L.
+   * On Intel, tWTRS/L should be left on auto and controlled with tWRRD_sg/dg respectively. Dropping tWRRD_dg by 1 will drop tWTRS by 1. Likewise with tWRRD_sg. Once they're as low as you can go, manually set tWTRS/L.
```

Comment:
tWTRS = tWRRD_dg + x
tWTRL = tWRRD_sg + x

Reply:
Ah yeah you're right, I looked at the order of them in AsRock and forgot they were linked _sg to _L and _dg to _S. My bad.